● snap.k8s.kubelet.service - Service for snap application k8s.kubelet
     Loaded: loaded (/etc/systemd/system/snap.k8s.kubelet.service; enabled; preset: enabled)
     Active: active (running) since Fri 2025-05-16 19:33:26 EDT; 3 days ago
   Main PID: 38157 (kubelet)
      Tasks: 19 (limit: 38302)
     Memory: 175.6M (peak: 185.6M)
        CPU: 9h 55min 49.735s
     CGroup: /system.slice/snap.k8s.kubelet.service
             └─38157 /snap/k8s/2500/bin/kubelet --anonymous-auth=false --authentication-token-webhook=true --authorization-mode=Webhook --cgroup-driver=systemd --client-ca-file=/etc/kubernetes/pki/client-ca.crt --cluster-dns=10.152.183.178 --cluster-domain=cluster.local --container-runtime-endpoint=/run/containerd/containerd.sock --containerd=/run/containerd/containerd.sock "--eviction-hard=memory.available<100Mi,nodefs.available<1Gi,imagefs.available<1Gi" --fail-swap-on=false --kubeconfig=/etc/kubernetes/kubelet.conf --node-ip=172.24.188.57 --node-labels=node-role.kubernetes.io/control-plane=,node-role.kubernetes.io/worker=,k8sd.io/role=control-plane --read-only-port=0 --register-with-taints= --root-dir=/var/lib/kubelet --serialize-image-pulls=false --tls-cert-file=/etc/kubernetes/pki/kubelet.crt --tls-cipher-suites=TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_RSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_256_GCM_SHA384 --tls-private-key-file=/etc/kubernetes/pki/kubelet.key

May 16 19:52:43 research21 k8s.kubelet[38157]: E0516 19:52:43.187706   38157 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"nova-spiceproxy\" with ImagePullBackOff: \"Back-off pulling image \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": ErrImagePull: [rpc error: code = DeadlineExceeded desc = failed to pull and unpack image \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to resolve reference \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to do request: Head \\\"https://registry.jujucharms.com/v2/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image/manifests/sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": dial tcp 185.125.188.51:443: i/o timeout, failed to pull and unpack image \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to resolve reference \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to do request: Head \\\"https://registry.jujucharms.com/v2/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image/manifests/sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": dial tcp: lookup registry.jujucharms.com on 127.0.0.53:53: server misbehaving]\"" pod="openstack/nova-0" podUID="0689787d-3910-4f26-84a4-d98f21f25a2b"
May 16 19:52:44 research21 k8s.kubelet[38157]: E0516 19:52:44.138429   38157 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"nova-spiceproxy\" with ImagePullBackOff: \"Back-off pulling image \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": ErrImagePull: [rpc error: code = DeadlineExceeded desc = failed to pull and unpack image \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to resolve reference \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to do request: Head \\\"https://registry.jujucharms.com/v2/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image/manifests/sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": dial tcp 185.125.188.51:443: i/o timeout, failed to pull and unpack image \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to resolve reference \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to do request: Head \\\"https://registry.jujucharms.com/v2/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image/manifests/sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": dial tcp: lookup registry.jujucharms.com on 127.0.0.53:53: server misbehaving]\"" pod="openstack/nova-0" podUID="0689787d-3910-4f26-84a4-d98f21f25a2b"
May 16 19:52:45 research21 k8s.kubelet[38157]: E0516 19:52:45.148735   38157 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"nova-spiceproxy\" with ImagePullBackOff: \"Back-off pulling image \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": ErrImagePull: [rpc error: code = DeadlineExceeded desc = failed to pull and unpack image \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to resolve reference \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to do request: Head \\\"https://registry.jujucharms.com/v2/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image/manifests/sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": dial tcp 185.125.188.51:443: i/o timeout, failed to pull and unpack image \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to resolve reference \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to do request: Head \\\"https://registry.jujucharms.com/v2/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image/manifests/sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": dial tcp: lookup registry.jujucharms.com on 127.0.0.53:53: server misbehaving]\"" pod="openstack/nova-0" podUID="0689787d-3910-4f26-84a4-d98f21f25a2b"
May 16 19:52:46 research21 k8s.kubelet[38157]: E0516 19:52:46.151333   38157 pod_workers.go:1301] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"nova-spiceproxy\" with ImagePullBackOff: \"Back-off pulling image \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": ErrImagePull: [rpc error: code = DeadlineExceeded desc = failed to pull and unpack image \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to resolve reference \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to do request: Head \\\"https://registry.jujucharms.com/v2/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image/manifests/sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": dial tcp 185.125.188.51:443: i/o timeout, failed to pull and unpack image \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to resolve reference \\\"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": failed to do request: Head \\\"https://registry.jujucharms.com/v2/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image/manifests/sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\\\": dial tcp: lookup registry.jujucharms.com on 127.0.0.53:53: server misbehaving]\"" pod="openstack/nova-0" podUID="0689787d-3910-4f26-84a4-d98f21f25a2b"
May 16 19:53:01 research21 k8s.kubelet[38157]: E0516 19:53:01.944143   38157 log.go:32] "PullImage from image service failed" err="rpc error: code = Unknown desc = failed to pull and unpack image \"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\": failed to resolve reference \"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\": pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed" image="registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9"
May 16 19:53:03 research21 k8s.kubelet[38157]: E0516 19:53:03.764398   38157 log.go:32] "PullImage from image service failed" err="rpc error: code = Unknown desc = failed to pull and unpack image \"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\": failed to resolve reference \"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\": pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed" image="registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9"
May 16 19:53:05 research21 k8s.kubelet[38157]: E0516 19:53:05.154538   38157 log.go:32] "PullImage from image service failed" err="rpc error: code = Unknown desc = failed to pull and unpack image \"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\": failed to resolve reference \"registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9\": pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed" image="registry.jujucharms.com/charm/cxpoxnvod1u9zuavqxnfmu8pseyhoa3d2np1q/nova-spiceproxy-image@sha256:40106c0a0b2972c865cf453956a00b3a32422971601f69092158b93c068503f9"
May 16 19:53:09 research21 k8s.kubelet[38157]: I0516 19:53:09.544230   38157 pod_startup_latency_tracker.go:104] "Observed pod startup duration" pod="openstack/nova-0" podStartSLOduration=35.872284464 podStartE2EDuration="9m2.544171339s" podCreationTimestamp="2025-05-16 19:44:07 -0400 EDT" firstStartedPulling="2025-05-16 19:44:39.85718021 -0400 EDT m=+672.994776897" lastFinishedPulling="2025-05-16 19:53:06.529067085 -0400 EDT m=+1179.666663772" observedRunningTime="2025-05-16 19:53:09.526163781 -0400 EDT m=+1182.663760481" watchObservedRunningTime="2025-05-16 19:53:09.544171339 -0400 EDT m=+1182.681768026"
May 16 20:03:27 research21 k8s.kubelet[38157]: E0516 20:03:27.338083   38157 log.go:32] "ExecSync cmd from runtime service failed" err="rpc error: code = DeadlineExceeded desc = failed to exec in container: timeout 1s exceeded: context deadline exceeded" containerID="66162b3a67c07afe628df37b7f2d106cdfe059e212c69336fe5d8e7af48a7f4d" cmd=["mongo","--port=37017","--tls","--tlsAllowInvalidHostnames","--tlsAllowInvalidCertificates","--tlsCertificateKeyFile=/var/lib/juju/server.pem","--eval","db.adminCommand('ping')"]
May 17 18:40:47 research21 k8s.kubelet[38157]: E0517 18:40:47.619801   38157 log.go:32] "ExecSync cmd from runtime service failed" err="rpc error: code = DeadlineExceeded desc = failed to exec in container: timeout 1s exceeded: context deadline exceeded" containerID="66162b3a67c07afe628df37b7f2d106cdfe059e212c69336fe5d8e7af48a7f4d" cmd=["mongo","--port=37017","--tls","--tlsAllowInvalidHostnames","--tlsAllowInvalidCertificates","--tlsCertificateKeyFile=/var/lib/juju/server.pem","--eval","db.adminCommand('ping')"]
